### ISSUE TITLE
SetBrightness Byte for revB

### ISSUE THREAD

**Issue by FabioZumbi12** [usausa/turing-smart-screen#2]
Hi,

What value do i need to use for set bright value for revB?
For revB class, we have this option in byte: `SetBrightness(byte level)`, but what value i need to use to set 0 from 1 of bright? I tryed some values and bytes converted but always turns the screen off.

Anyway, if you could convert the input value of this method to int like the class of revA, will help a lot.

Thanks!

**Comment by usausa** [comment]
Because RevB supports different brightness ranges for different versions, the values are directly specified without conversion.
I am also considering an abstraction that would treat RevA and RevB the same way, but at this stage, please keep the conversion on the caller side.

**Comment by FabioZumbi12** [comment]
Yeah, i am using a dynamic object to store the rev classes inside my class helper as the methods are equals, the only difference is SetBrightness, but no problem for now!

```
private dynamic display;

        public DisplayTools(string rev, string port)
        {
            _rev = rev; 
            if (rev == "a")
                display = new TuringSmartScreenRevisionA(port);
            else
                display = new TuringSmartScreenRevisionB(port);
        }
```

**Comment by usausa** [comment]
I added new abstraction API in v2.0.0, please try it.
The Brightness input for the new API is 0-100.
Please refer to README.md and Example project that have added for usage.
